src/ScriptBuilder/Outbox/Create_MsSqlServer.sql

```diff
@@ -32,23 +32,6 @@ begin
   declare @createDispatchedAtIndex nvarchar(max);
   set @createDispatchedAtIndex = '
   create index Index_DispatchedAt
-  on ' + @tableName + '(DispatchedAt);';
+  on ' + @tableName + '(DispatchedAt) where Dispatched = 1;';
```

Comment:
Ok nevermind, I guess it was [introduced in SQL 2008](https://www.mssqltips.com/sqlservertip/1785/sql-server-filtered-indexes-what-they-are-how-to-use-and-performance-advantages/) and the Microsoft link with its **SQL Server (starting with 2016)** label at the top is full of it.